### PR TITLE
No user robust tracking

### PR DESCRIPTION
**Pull request by gfiorav**
Please CR, @javitonino !

### CODE REVIEW

File: app/controllers/carto/api/visualization_exports_controller.rb [hunk 1]
```diff
@@ -47,7 +47,7 @@ def create
         download_path = CartoDB.path(self, 'visualization_export_download', download_path_params)
         Resque.enqueue(Resque::ExporterJobs, job_id: visualization_export.id, download_path: download_path)
 
-        Carto::Tracking::Events::ExportedMap.new(current_user, @visualization).report
+        Carto::Tracking::Events::ExportedMap.new(current_viewer, @visualization).report if @current_viewer
```
Comment (javitonino) [inline app/controllers/carto/api/visualization_exports_controller.rb]: `current_viewer` vs `@current_viewer`. I think this never evaluates as `@current_viewer` is not set. Also, no alternative tracking for anonymous exports?

Comment (gfiorav) [inline app/controllers/carto/api/visualization_exports_controller.rb]: typo

Comment (gfiorav) [inline app/controllers/carto/api/visualization_exports_controller.rb]: no, anonymous downloads will not be tracked!

File: lib/carto/tracking/events.rb [hunk 2]
```diff
@@ -48,19 +64,9 @@ def report
...
-            username: @user ? @user.username : nil,
-            email: @user ? @user.email : nil,
-            plan: @user ? @user.account_type : nil,
-            user_active_for: user_age_in_days_with_decimals,
-            user_created_at: user_created_at,
-            organization: @user && @user.organization_user? ? @user.organization.name : nil,
-            event_origin: 'Editor',
-            creation_time: now
-          }
+          properties = user_properties(@user, now: now)
```
Comment (javitonino) [inline lib/carto/tracking/events.rb]: Cool refactor :sparkles:

**Comment by javitonino** [comment]
After the latest commit: :+1:

**Pull request merged**